ci-operator/jobs/redhat-developer/gitops-operator/redhat-developer-gitops-operator-v1.9-presubmits.yaml

```diff
@@ -298,12 +303,18 @@ presubmits:
           readOnly: true
       serviceAccountName: ci-operator
       volumes:
+      - name: boskos
+        secret:
+          items:
+          - key: credentials
+            path: credentials
+          secretName: boskos-credentials
```

Comment:
what is this `boskos` creds that we are adding?

Reply:
This was added automatically, must be related to the test-profile change